msteams-platform/tutorials/code-samples.md

```diff
@@ -3,7 +3,7 @@ title: Microsoft Teams code samples
 description: Links and descriptions of sample applications for the Microsoft Teams developer platform
 keywords: Microsoft Teams developer samples
 ---
-# Tutorials and code samples for the Microsoft Teams developer platform
+# Tutorials for the Microsoft Teams developer platform
 
 Here you will find a list of tutorials and code samples that demonstrate how you can extend the Teams developer platform capabilities by creating custom apps.
```

Comment:
Line 8, please change to:

Here you will find a list of tutorials that demonstrate how you can extend the Teams developer platform capabilities by creating custom apps.